gem_city.rb

```diff
@@ -0,0 +1,50 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  def initialize
+    @people = { thieves: 5, Officers: 1 }
```

Comment:
If you make these into instance variables, you can clean up a lot of your code.

https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby
http://ruby-doc.org/core-2.0.0/Module.html#method-i-attr_accessor